doc/developer-guides/hld/virtio-net.rst

```diff
@@ -484,7 +484,7 @@ optional):
 
 .. code-block:: none
 
-    -s 4,virtio-net,<tap_name>,[mac=<XX:XX:XX:XX:XX:XX>]
+    -s 4,virtio-net,<tap=name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Comment:
I think this is a better representation of the format:
```suggestion
    -s 4,virtio-net,tap=<name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Reply:
I think it better to follow the current format.

Reply:
Done